library/smarty/plugins/function.headerTemplate.php

```diff
@@ -13,9 +13,10 @@
  * of the License, or (at your option) any later version.
```

Comment:
I removed that file, remnants of the beginnings of this project while I was still working stuff out